docs/installing_agones.md

```diff
@@ -0,0 +1,208 @@
+# Install and configure Agones on Kubernetes
+
+In this quickstart, we will create a Kubernetes cluster, and populate it with the resource types that power Agones.
+
+# Table of contents
+
+1. [Setting up a Google Kubernetes Engine (GKE) cluster](#setting-up-a-google-kubernetes-engine-gke-cluster)
+   1. [Before you begin](#before-you-begin)
+   1. [Choosing a shell](#choosing-a-shell)
+      1. [Cloud shell](#cloud-shell)
+      1. [Local shell](#local-shell)
+   1. [Creating the cluster](#creating-the-cluster)
+      1. [Creating the firewall](#creating-the-firewall)
+1. [Setting up a Minikube cluster](#setting-up-a-minikube-cluster)
+   1. [Installing Minikube](#installing-minikube)
+   1. [Creating an agones profile](#creating-an-agones-profile)
+   1. [Starting Minikube](#starting-minikube)
+1. [Enabling creation of RBAC resources](#enabling-creation-of-rbac-resources)
+1. [Installing Agones](#installing-agones)
+   1. [Confirming Agones started successfully](#confirming-agones-started-successfully)
+1. [What's next](#whats-next)
+
+# Setting up a Google Kubernetes Engine (GKE) cluster
+
+Follow these steps to create a cluster and install Agones directly on Google Kubernetes Engine (GKE).
+
+## Before you begin
+
+Take the following steps to enable the Kubernetes Engine API:
+
+1. Visit the [Kubernetes Engine][kubernetes] page in the Google Cloud Platform Console.
+1. Create or select a project.
+1. Wait for the API and related services to be enabled. This can take several minutes.
+1. [Enable billing][billing] for your project.
```

Comment:
Stealing from the [istio docs](https://istio.io/docs/setup/kubernetes/quick-start-gke-dm.html), shall we add:

> If you are not an existing GCP user, you may be able to enrol for a $300 US [Free Trial](https://cloud.google.com/free/) credit.

Just so that it's clear you can use the free trial :+1:

Reply:
Done :)